streaming/src/main/scala/org/apache/spark/streaming/scheduler/JobScheduler.scala

```diff
@@ -179,7 +181,14 @@ class JobScheduler(val ssc: StreamingContext) extends Logging {
       PairRDDFunctions.disableOutputSpecValidation.withValue(true) {
         job.run()
       }
+      ssc.sc.setLocalProperty(JobScheduler.BATCH_TIME_PROPERTY_KEY, null)
+      ssc.sc.setLocalProperty(JobScheduler.OUTPUT_OP_ID_PROPERTY_KEY, null)
       eventActor ! JobCompleted(job)
     }
   }
 }
+
+private[streaming] object JobScheduler {
+  private[streaming] val BATCH_TIME_PROPERTY_KEY = "spark.streaming.internal.batchTime"
```

Comment:
If the object private[streaming] then all the field are already private[streaming], isnt it?